src/to-contain-html.js

```diff
@@ -0,0 +1,24 @@
+import {matcherHint, printReceived} from 'jest-matcher-utils'
+import {checkHtmlElement} from './utils'
+
+export function toContainHtml(container, htmlText) {
```

Comment:
This currently does not match the capitalization in the typescript definition. I suggest renaming everything to `toContainHTML`.